src/core/config/descriptor_providers/xlio_config_schema.json

```diff
@@ -0,0 +1,1454 @@
+{
```

Comment:
Do we have somewhere a flat list of all available parameters?
Something like
core.resources.memory_limit
core.resources.hugepages
...

If there is no such list maybe worth to add in the description the full 'path' of the variable as it is very hard to track the hierarchy

Reply:
Yes, see src/core/config/README.md and src/core/config/mappings.py :)